osu.Game/Screens/Edit/EditorScreen.cs

```diff
@@ -49,5 +50,54 @@ protected override void PopOut()
             this.ScaleTo(0.98f, 200, Easing.OutQuint)
                 .FadeOut(200, Easing.OutQuint);
         }
+
+        #region Clipboard operations
+
+        public BindableBool CanCut { get; } = new BindableBool();
+
+        /// <summary>
+        /// Performs a "cut to clipboard" operation appropriate for the given screen.
+        /// </summary>
+        protected virtual void PerformCut()
```

Comment:
Wondering if it makes sense for the clipboard instance to be passed through here (and the other methods in this file) as a parameter? Should theoretically avoid any need to DI the clipboard.

Reply:
It's a little dicey here because the screens will `BindValueChanged` to the clipboard content to update the availability state of all the actions, so I'd still probably feel more comfortable with DI'ing.

Reply:
Unless I split `CanCut()` into methods and call that on clipboard content changes, but that feels less flexible than what is currently here (especially so that the action availability may depend on other context, like e.g. how many objects are selected - which the composer already uses).

Reply:
I'm not too fussed just yet. Let's go with what is here and see how it builds out.